angular/apis/etsy/gymnastics.js

```diff
@@ -0,0 +1,23 @@
+var data = require('./trending.json')
+var _ = require('lodash');
+var trending = data.results;
+
+
+function keyChecker(key){
+	return _.map(trending,key)
+};
```

Comment:
I have no idea what this function does. Might be nice to document it... or just write it as a series of tests instead.